src/frontend/src/components/RecordSearch/Layout/index.tsx

```diff
@@ -38,6 +40,15 @@ export default function Layout({
   const panelClass = "bg-white shadow br3 ";
   const panelHeading = "f5 fw7 tc mt2 mb3";
 
+  useEffect(setUpScrollSpy, [isExpandedView]);
+
+  useEffect(() => {
+    document.querySelector("html")?.classList.toggle("smooth-scroll");
+    return () => {
+      document.querySelector("html")?.classList.toggle("smooth-scroll");
+    };
+  });
```

Comment:
What does this do? Why not have it always smooth scrolling for example?

Reply:
I wanted to have the smooth scrolling on for the right sidebar case/charge navigation links. But keeping that on for the entire site did not seem right when doing page transitions via various links which use `window.scrollTo(0, 0)` in their `onClick` handlers. Because of that, I remove the smooth scroll when unmounting the Layout component.

Reply:
Thanks for the explanation!